index.html

```diff
@@ -4197,16 +4197,20 @@ <h3>Securing Mechanism Specifications</h3>
         </p>
 
         <p>
-Securing mechanism specifications MUST provide a verification mechanism that
+Securing mechanism specifications MUST provide a verification algorithm that
 returns only the information in the [=conforming document=] that has been
 secured, without any securing mechanism information included, such as `proof` or
-JOSE/COSE header parameters and signatures. Specifications MAY provide additional mechanisms to convey
-other information that might be helpful (for example, during validation or for
-debugging purposes), such as securing mechanism data. A securing mechanism's
-verification algorithm MUST provide an interface that receives inputs of a media
-type ([=string=] |inputMediaType|) paired with either a sequence of bytes
-([=byte sequence=] |inputBytes|) or a document ([=map=] |inputDocument|), and
-returns a verification result with at least the following [=struct/items=]:
+JOSE/COSE header parameters and signatures. Verification algorithms MAY return
+additional information that might be helpful (for example, during validation or
+for debugging purposes), such as securing mechanism details. A verification
+algorithm MUST provide an interface that receives a media type ([=string=]
+|inputMediaType|) and secured data ([=byte sequence=] or [=map=] |securedData|).
+If the verification algorithm expects a [=byte sequence=], provide |securedData|
+to the algorithm. If the verification algorithm expects a [=map=], provide the
+result of [=parsing JSON bytes to an Infra value=] given |securedData|. Securing
```

Comment:
Maybe "input data" instead of "securedData" since this algorithm makes the "securing determination"? I can live with it either way (or with some other name), just drawing attention here.

```suggestion
|inputMediaType|) and input data ([=byte sequence=] or [=map=] |inputData|).
If the verification algorithm expects a [=byte sequence=], provide |inputData|
to the algorithm. If the verification algorithm expects a [=map=], provide the
result of [=parsing JSON bytes to an Infra value=] given |inputData|. Securing
```